libs/s25main/world/GameWorldView.h

```diff
@@ -38,11 +38,11 @@ class GameWorldView
     std::vector<IDrawNodeCallback*> drawNodeCallbacks;
 
     /// Show building quality icons
-    bool show_bq;
+    bool& show_bq;
```

Comment:
this is so extremly implicit/intransparent what it changes. Can we have a better way? i.e directly use the SETTINGS then?

Reply:
Agreed. Maybe even better: Where this is created, also set the options after creation from outside and when removed save it to SETTINGS?
I imagine having multiple of these instances for split-screen somewhere in the future.

Reply:
Makes sense, I will change this, with possible future splitscreen in mind

Reply:
I changed this to retrieve the values from SETTINGS in the constructor of the GameWorldView and write them back to SETTINGS in the destructor.